### ISSUE TITLE
The final point or the point to cancel mission doesn't work

### ISSUE THREAD

**Issue by Alexbello3** [ElusionPDX/Delboys#1]
Hello,i just install this on my server.But when i spawn the truck and the trailer and finish the mission.And I go to the "final" point o the deleter mission point doesn't work.Can you repair it?Thanks

**Comment by ElusionPDX** [comment]
Need more info then this please. Did the trailer get removed at the waypoint.

**Comment by Alexbello3** [comment]
Yes the trailer get removed in the destination.But the truck no.

**Comment by ElusionPDX** [comment]
OK it doesn't remove the truck as the mission it already done as you dropped off the trailer already. That marker is to cancel the mission if you dont wanna drive that far or just started a mission by mistake.   

Will push a update to it in a few days that will change some things up. Just keep an eye out.

**Comment by Alexbello3** [comment]
Yes,that marker that you say it doesn't work either.But thanks.I will be careful

**Comment by ElusionPDX** [comment]
it only cancels the mission.   I didnt make this job im just keeping it alive as i like the job.  Updates will include proper vehicle deleter

**Comment by Alexbello3** [comment]
Yes yes,good job and good idea!
